include/miral/miral/append_event_filter.h

```diff
@@ -26,13 +26,26 @@ namespace mir { class Server; }
 
 namespace miral
 {
+/// Appends an event filter to the Mir server event pipeline.
+///
+/// Filters are processed in order. This filter is appended **after** any existing filters,
+/// including those provided by [miral::WindowManagementPolicy].
+///
+/// An event is passed to this filter only if no earlier filter has already handled it.
+///
+/// See also:
+/// - AppendKeyboardEventFilter - a specialized event filter which only filters keyboard events
+///
+/// \sa AppendKeyboardEventFilter
```

Comment:
There's double "See also" in the result, we should decide on one way?
<img width="1012" height="226" alt="image" src="https://github.com/user-attachments/assets/6a7d9348-ccbf-4e61-888e-25f6df98e209" />

Reply:
> most IDEs won't see the doxygen-style "See also"

That sounds like a broken IDE to me… if anything, I would expect the Doxygen stuff to be parsable.

It is a bit heavy when there's more than one… but that's breathe/exhale's fault - maybe something we can fix when multiple `/sa`s exist atop one other.